modules/distribution-workflowoperation/src/main/java/org/opencastproject/workflow/handler/distribution/PublishEngageWorkflowOperationHandler.java

```diff
@@ -114,7 +114,7 @@
   private static final String MERGE_FORCE_FLAVORS_DEFAULT = "dublincore/*,security/*";
 
   /** The default path to the player **/
-  protected static final String DEFAULT_PLAYER_PATH = "/engage/ui/watch.html";
+  protected static final String PLAYER_PATH = "/watch/";
```

Comment:
It would help to clarify in the comment that it reference an Engage UI REST Endpoint that returns a path to the player at runtime. Or, as @JulianKniephoff  suggested, provide a common service that returns the common player redirect path.